lib/src/logic.dart

```diff
@@ -100,9 +100,15 @@ class Logic {
   BigInt get valueBigInt => value.toBigInt();
 
   /// Returns `true` iff the value of this signal is valid (no `x` or `z`).
+  /// 
+  /// Throws an exception 
+  @Deprecated('Use currentValue.isValid instead.')
```

Comment:
Should say "Use value.isValid instead." since `_currentValue` is a private variable (starting with an underscore `_`)

Reply:
This still needs to be updated to "value.isValid" in the `Deprecated` note